app/lib/index.js

```diff
@@ -402,12 +402,18 @@ app.openFiles = function(filePaths) {
  */
 app.createEditorWindow = function() {
 
-  const nodeIntegration = !!flags.get('dangerously-enable-node-integration');
+  const nodeIntegration = flags.get('dangerously-enable-node-integration', false);
 
   if (nodeIntegration) {
     log.warn('nodeIntegration is enabled via --dangerously-enable-node-integration');
   }
 
+  const contextIsolation = !flags.get('dangerously-disable-context-isolation', false);
+
+  if (contextIsolation) {
+    log.warn('contextIsolation is disabled via --dangerously-disable-context-isolation');
+  }
```

Comment:
The warning is dispatched when the flag is __not__ set.

```suggestion
  if (!contextIsolation) {
    log.warn('contextIsolation is disabled via --dangerously-disable-context-isolation');
  }
```